docs/FAQ.md

```diff
@@ -470,3 +471,24 @@ later) finishes before the slow task that triggered speculation. If the speculat
 finishes first then that's good, it is working as intended. If many tasks are speculating, but the
 original task always finishes first then this is a pure loss, the speculation is adding load to
 the Spark cluster with no benefit.
+
+### Why is my query in GPU mode slower than CPU mode?
+
+Below are some troubleshooting tips on GPU query performance issue:
+* Make sure the query in GPU mode is fully on GPU. Please refer to
+  [Getting Started on Spark workload qualification](./get-started/getting-started-workload-qualification.md)
+  for more details. If there are some CPU fallbacks, check if those are some known features which
+  can be enabled by turning on some Spark RAPIDS parameters. If the features needed do not exsit in
+  the most recent release of Spark RAPIDS, please file 
+  [feature request](https://github.com/NVIDIA/spark-rapids/issues) with a minimum reproduce.
+
+* Tune the Spark and Spark RAPIDS parameters such as `spark.sql.shuffle.partitions`, 
+  `spark.sql.files.maxPartitionBytes` and `spark.rapids.sql.concurrentGpuTasks` to get the best run.
+  Please refer to [Tuning Guide](./tuning-guide.md) for more details.
+
+* Identify the most time consuming part of the query. You can use 
+  [Profiling tool](./spark-profiling-tool.md) to process the spark eventlog to get more insights of 
+  the query performance. For example, if IO is the bottleneck, we suggest optimize the backend 
+  storage IO performance. It's because the most suitable query type is computation bound instead of 
+  IO or Network bound.
```

Comment:
```suggestion
  the query performance. For example, if I/O is the bottleneck, we suggest optimizing the backend 
  storage I/O performance because the most suitable query type is computation bound instead of 
  I/O or network bound.
```